WordPress/Classes/ViewRelated/Me/MeViewController.swift

```diff
@@ -414,6 +414,24 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
         headerView.onGravatarPress = { [weak self] in
             self?.presentGravatarPicker()
         }
+        headerView.onDroppedImage = { [weak self] image in
+            let imageCropViewController = ImageCropViewController(image: image)
+            imageCropViewController.maskShape = .square
+            imageCropViewController.shouldShowCancelButton = true
+
+            imageCropViewController.onCancel = { () -> Void in
```

Comment:
Was this change accidentally reverted when fixing merge conflicts?

Reply:
After merging from develop `_ in` caused a compile error: `Cannot assign value of type '(_) -> ()' to type '(() -> Void)?'` so I reverted it. I missed an earlier comment that it needed `[weak self] in` instead of `_ in`. Corrected